catalog/nsili/catalog-nsili-sourcestoquery-ui/pom.xml

```diff
@@ -108,7 +108,7 @@
         <dependency>
             <groupId>org.webjars.bower</groupId>
             <artifactId>moment</artifactId>
-            <version>2.18.1</version>
+            <version>2.20.1</version>
```

Comment:
✏️ Maybe pull this into a version property so that `catalog-nsili-sourcestoquery-ui` and `video-admin-plugin` use the same version

Reply:
Right. Good idea